docs/AboutUs.md

```diff
@@ -15,40 +14,49 @@ You can reach us at the email `e0406389@u.nus.edu`
 
 [[github](https://github.com/JayChenYJ)]
 [[portfolio](team/jaychenyj.md)]
+You can reach me at the email `e0406389@u.nus.edu`
```

Comment:
put a spacing between portfolio and this sentence, so will be in a new paragraph :)